test/e2e/network_test.go

```diff
@@ -134,7 +134,9 @@ func (tc *testContext) testEastWestNetworking(t *testing.T) {
 			}
 			require.NoError(t, err, "could not create Windows Server deployment")
 			defer tc.deleteDeployment(winServerDeployment.Name)
-			tc.collectDeploymentLogs(winServerDeployment)
+			if err := tc.collectDeploymentLogs(winServerDeployment); err != nil {
+				log.Printf("error collecting deployment logs: %s", err)
```

Comment:
`%v` if you log `err` directly. alternatively `err.Error()` with `%s`

applies throughout

Reply:
What is the functional difference given the error interface implements `String()`?

Reply:
for some reason i thought this would result in a type error when called, tested it out and found no issue